front/pages/api/w/[wId]/invitations/index.ts

```diff
@@ -152,7 +158,48 @@ async function handler(
         });
       }
       const existingMembers = await getMembers(auth);
-
+      const unconsumedInvitations = await MembershipInvitation.findAll({
```

Comment:
Let's move this to the `lib/api/*` (see [thread](https://dust4ai.slack.com/archives/C050SM8NSPK/p1712230956371889))